catalyst-gateway/bin/src/settings/mod.rs

```diff
@@ -128,6 +144,22 @@ struct EnvVars {
     /// The Address of the Event DB.
     event_db_password: Option<StringEnvVar>,
 
+    /// Sets the maximum number of connections managed by the pool.
+    /// Defaults to 10.
+    event_db_max_size: u32,
```

Comment:
Maybe it would be better to name it something like `event_db_max_connections`? It is clear what `max_size` does when it is called during the pull construction, but "db max size" sounds confusing to me.

Reply:
yes I agree, changing now